plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -102,10 +102,14 @@ class ContainerServiceFormSection extends Component {
         </a>.
       </Trans>
     );
+    const { data, path } = this.props;
```

Comment:
This is complex enough to warrant a `isDockerContainer()` helper method on the class. It would make the render function read much better in my opinion.